src/printer/tptp/tptp_printer.cpp

```diff
@@ -61,6 +54,34 @@ void TptpPrinter::toStream(std::ostream& out, const smt::Model& m) const
       << endl;
 }
 
+void TptpPrinter::toStreamCmdSuccess(std::ostream& out) const
+{
+  getPrinter(Language::LANG_SMTLIB_V2_6)->toStreamCmdSuccess(out);
```

Comment:
We use the `Language::LANG_SMTLIB_V2_6` printer for TPTP?

Reply:
Yeah, our TPTP printer doesn't really do much...